### PR TITLE
FIX illegal_extensions in unit tests not being honored

### PR DESCRIPTION
**Pull request by mikenz**

### CODE REVIEW

File: src/Dev/SapphireTest.php [hunk 1]
```diff
@@ -356,15 +356,15 @@ public static function setUpBeforeClass()
                 continue;
             }
             foreach ($extensions as $extension) {
-                if (!class_exists($extension) ||$class::has_extension($extension)) {
+                if (!class_exists($extension) || !$class::has_extension($extension)) {
```
Comment (mikenz) [inline src/Dev/SapphireTest.php]: This is the important change. The rest is just whitespace.

**Comment by dhensby** [comment]
I think the test failure may well be related?

**Comment by mikenz** [comment]
The Travis failure is just this: https://github.com/silverstripe/silverstripe-versioned/pull/6/files

**Comment by dhensby** [comment]
@mikenz ah - I've merged that and restarted the tests

**Comment by dhensby** [comment]
Looks like the CMS test isn't getting on to well :/

**Comment by mikenz** [comment]
Oh, I've fixed those locally - I'll create a PR for them too

**Comment by mikenz** [comment]
Fix CMS unit tests https://github.com/silverstripe/silverstripe-cms/pull/1771

**Comment by dhensby** [comment]
Right, I seemed to have missed the PR for cms when I did the work last night- I've opened that here: https://github.com/silverstripe/silverstripe-cms/pull/1772 and it looks like the tests will pass.

This PR looks good

**Pull request merged**